src/update-feeds-loops/contracts.ts

```diff
@@ -51,9 +51,9 @@ export const decodeGetBlockNumberResponse = Number;
 export const decodeGetChainIdResponse = Number;
 
 export interface Beacon {
-  airnodeAddress: AirnodeAddress;
-  templateId: TemplateId;
-  beaconId: string;
+  airnodeAddress: Address;
+  templateId: Hex;
```

Comment:
[Keccak256Hash](https://github.com/api3dao/commons/blob/fb07e2c083cf63bc35d500aca37eb3fb7ee411c8/src/blockchain-utilities/schema.ts#L26C13-L26C26) for beacon and template Ids?